platform/ios/ios.xcodeproj/project.pbxproj

```diff
@@ -2899,6 +2912,7 @@
 			baseConfigurationReference = 55D8C9941D0F133500F42F10 /* config.xcconfig */;
 			buildSettings = {
 				BITCODE_GENERATION_MODE = bitcode;
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "";
```

Comment:
why is this set to an empty string?

Reply:
This is Xcode’s way of disabling code signing. 🤷‍♂️